charts/replicated-library/README_CHANGELOG.md.gotmpl

```diff
@@ -10,6 +10,13 @@ All notable changes to this library Helm chart will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [0.7.0]
+### Changed
+
+- BREAKING: The `appName` key for services is now an optional list of apps to associate the service to.
```

Comment:
```suggestion
- BREAKING: The `appName` key for services is now an optional list instead of a string. Charts using the previous implementation will need to convert the string into a single entry list which will work as before.
```

Reply:
I'd like, when we can, to see if we can highlight what a chart author needs to do about a breaking change and just that it changed. Someone who hasn't touched a chart in a few months needs to understand quickly what they need to do to accommodate.

P.S. While I added this as a recommendation if you accept it you'll have to re-run the `helm-docs` command to update it everywhere it's used.